shared/chat/conversation/messages/wrapper/index.js

```diff
@@ -43,6 +43,7 @@ export type Props = {|
   authorIsAdmin: ?boolean,
   authorIsOwner: ?boolean,
   centeredOrdinal: boolean,
+  centeredOrdinalHighlightMode: Types.CenterOrdinalHighlightMode,
```

Comment:
Could you drive this without the additional `centeredOrdinal` prop? The container can always pass in `none` if this isn't the right ordinal

Reply:
Sure